### PR TITLE
bulid blog aggregator as a tutorial

### PR DESCRIPTION
**Pull request by wcwagner22**
Rough Draft, copied the information into another .md file similar to the other python tutorial we have.

### CODE REVIEW

File: content/tutorials/build-blog-aggregator.md [hunk 1]
```diff
@@ -0,0 +1,735 @@
...
+date: "2024-09-17"
+categories:
+  - "tutorials"
+  - "python"
+images:
+  - /https://storage.googleapis.com/qvault-webapp-dynamic-assets/course_assets/EqY2AKt.png
+toc: true
+---
+
+# Welcome
```
Comment (wagslane) [inline content/tutorials/build-blog-aggregator.md:13]: Reevaluate all these headings:

1. Each step title should be an h2 element (2 pounds)
2. Step stub headings should be h3s or h4s as appropriate (its a tree structure)

File: content/tutorials/build-blog-aggregator.md [hunk 2]
```diff
@@ -0,0 +1,735 @@
+---
+title: "Build a Blog Aggregator"
+author: Lane Wagner
+date: "2024-09-17"
+categories:
+  - "tutorials"
+  - "python"
```
Comment (wagslane) [inline content/tutorials/build-blog-aggregator.md:7]: This is a Go project

**Review by wagslane — CHANGES_REQUESTED** [review]

File: content/tutorials/build-blog-aggregator.md [hunk 3]
```diff
@@ -0,0 +1,735 @@
...
+images:
+  - /https://storage.googleapis.com/qvault-webapp-dynamic-assets/course_assets/EqY2AKt.png
```
Comment (wagslane) [inline content/tutorials/build-blog-aggregator.md:9]: use the image-min script and add this as a file in the project. Winston should be able to help with this

```
npm run image-min
```

**Comment by wcwagner22** [comment]
Had some issues with committing this one, I was updated on the branch but it kept telling me I was outdated. Did a force, lmk if I messed anything up

**Review by wagslane — APPROVED** [review]

**Pull request merged**